azext_edge/edge/_help.py

```diff
@@ -1216,3 +1273,66 @@ def load_iotops_help():
             --sa-resource-id $STORAGE_ACCOUNT_RESOURCE_ID --sa-container myschemacontainer
             -l westus2 --desc 'Contoso factory X1 schemas' --display-name 'Contoso X1' --tags env=prod
     """
+
+    helps[
+        "iot ops schema version"
+    ] = """
+        type: group
+        short-summary: Schema version management.
+        long-summary: |
+          A schema version contains the schema content associated with that version.
+    """
+
+    helps[
+        "iot ops schema version show"
+    ] = """
+        type: command
+        short-summary: Show details of a schema version.
+        examples:
+        - name: Show details of target schema version 1.
+          text: >
+            az iot ops schema version show --name 1 --schema myschema --registry myregistry -g myresourcegroup
+    """
+
+    helps[
+        "iot ops schema version list"
+    ] = """
+        type: command
+        short-summary: List schema versions for a specific schema.
+        examples:
+        - name: List all schema versions for the schema 'myschema' in the schema registry 'myregistry'.
+          text: >
+            az iot ops schema list -g myresourcegroup --registry myregistry --schema myschema
+    """
+
+    helps[
+        "iot ops schema version remove"
+    ] = """
+        type: command
+        short-summary: Remove a target schema version.
+        examples:
+        - name: Remove schema version 1.
+          text: >
+            az iot ops schema version remove -n 1 -g myresourcegroup --registry myregistry --schema myschema
+    """
+
+    helps[
+        "iot ops schema version add"
+    ] = """
+        type: command
+        short-summary: Add a schema version to a schema.
+        long-summary: |
+                      To add a version, the associated storage account will need to have public network access enabled.
+                      For more information on the delta file format, please see
+                      https://github.com/delta-io/delta/blob/master/PROTOCOL.md#column-mapping and
+                      https://learn.microsoft.com/en-us/fabric/data-engineering/lakehouse-and-delta-tables. (#TODO AKA.ms)
```

Comment:
will need to make aka.ms links for these two - wanted opinions on names:
aka.ms/delta-protocol
aka.ms/lakehouse-delta-example

Reply:
let's stick to just the msft one for now, I think the aka.ms link is fine